### PR TITLE
Release fix

### PR DESCRIPTION
**Pull request by mkuznyetsov**
### What does this PR do?
Fix some issues with release script discovered during the latests releases
Remove unused cico_script

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)

- [x] [The Eclipse Contributor Agreement is valid](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-eclipse-contributor-agreement-is-valid)
- [x] [Code produced is complete](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-produced-is-complete)
- [x] [Code builds without errors](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-builds-without-errors)
- [x] [Tests are covering the bugfix](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#tests-are-covering-the-bugfix)
- [x] [The repository devfile is up to date and works](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-repository-devfile-is-up-to-date-and-works)
- [x] [Sections `What issues does this PR fix or reference` and `How to test this PR` completed](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#sections-what-issues-does-this-pr-fix-or-reference-and-how-to-test-this-pr-completed)
- [x] [Relevant user documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [x] [Relevant contributing documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [x] [CI/CD changes implemented, documented and communicated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#cicd-changes-implemented-documented-and-communicated)

### Reviewers

Reviewers, please comment how you tested the PR when approving it.

### Happy Path Channel

HAPPY_PATH_CHANNEL=stable

### CODE REVIEW

**Review by benoitf — APPROVED** [review]

**Comment by benoitf** [comment]
can we remove all cico files. Maybe another PR ?

cico_build_master.sh
cico_build_nightly.sh
cico_build_pr.sh
cico_build_release.sh
cico_common.sh

@azatsarynnyy are we using those files in github actions ?

**Comment by azatsarynnyy** [comment]
> can we remove all cico files. Maybe another PR ?
> 
> cico_build_master.sh
> cico_build_nightly.sh
> cico_build_pr.sh
> cico_build_release.sh
> cico_common.sh
> 
> @azatsarynnyy are we using those files in github actions ?

I've created the corresponding issue https://github.com/eclipse/che/issues/19100 to get rid of the PR check and all the related scripts.

**Review by azatsarynnyy — COMMENTED** [review]
Don't we need [`AKAMAI_CHE_AUTH` secret](https://github.com/eclipse/che-theia/blob/5b151b0bc2aef303748f96b88b0e2afb89c26200/cico_common.sh#L17) set for the release workflow?

**Comment by nickboldt** [comment]
@azatsarynnyy excellent question. I have a properties file I'm using for CRW's akamai publishing of crw-theia. I've uploaded the base64-encoded version of that properties file as a new secret AKAMAI_CHE_AUTH into both eclipse/che-theia and eclipse/che-release so it's available in both. 

Looks like if the akamai stuff happens, we should see `push-cdn-files-to-akamai.sh` run, and then NOT see this in the console log:

```
echo "CDN files will not be pushed to the Akamai directory since the 'AKAMAI_CHE_AUTH' environment variable is not set"
```

If you see any other missing secrets, let me know and I can upload 'em. I've also sent copies of that key to Florent and Mykhailo so we have backups of it. If you email me your public gpg key, I can also forward you a copy.

File: .github/workflows/release.yml
Comment (nickboldt) [inline .github/workflows/release.yml:70]: I suppose we also need to add `AKAMAI_CHE_AUTH` to the env: section below to expose it from secret -> env var?

**Comment by nickboldt** [comment]
@benoitf is the point of the `PR Checklist` that we check them all off before approving PRs? Just wondering -- if they're not being used, what's the value of including 'em?

**Comment by benoitf** [comment]
@nickboldt all items are there for contributors to ensure they're following the guidelines.
@mkuznyetsov should check them before merging but it's a PR having all checks passing. I think we didn't have that for months (especially the happy path jobs) and even the ci.centos job ! so this is why I approved

**Pull request merged**
